Makefile

```diff
@@ -85,7 +85,7 @@ endif
 
 all: cephcsi
 
-.PHONY: go-test static-check mod-check go-lint lint-extras commitlint codespell
+.PHONY: go-test static-check mod-check go-lint lint-extras commitlint codespell trivvy
```

Comment:
maybe add a functional `trivvy` target too? Or drop this from the makefile.

Reply:
dropped for now